cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -410,6 +410,50 @@ DROP ALIAS ... [IF EXISTS] FOR DATABASE
 a|
 New Cypher command for dropping a database alias.
 
+
+a|
+label:functionality[]
+label:new[] +
+New privilege:
+[source, cypher, role="noheader"]
+----
+ALIAS MANAGEMENT
+----
+a|
+New privilege that allows a user to create, modify and delete aliases. Only supported since version 4.4.6.
```

Comment:
What about the SHOW ALIAS privilege? It should be included both in this text and as its own entry.